Formula/djl-serving.rb

```diff
@@ -5,6 +5,12 @@ class DjlServing < Formula
   sha256 "f87189733657e80c5f3ad94a11eb73f3bb7a3433f2fbd5441662a94e0d887f9c"
   license "Apache-2.0"
 
+  # The official release notes would be in `deepjavalibrary/djl` repo
```

Comment:
```suggestion
  # `djl-serving` versions aren't considered released until a corresponding
  # release is created in the main `deepjavalibrary/djl` repository.
```

This wording may explain the situation a bit more clearly (with respect to the linked issue).